src/shared/cache/GenomeNexusCache.ts

```diff
@@ -3,13 +3,14 @@ import {
     extractGenomicLocation,
     genomicLocationString,
 } from 'shared/lib/MutationUtils';
-import { VariantAnnotation } from 'cbioportal-frontend-commons';
+import { VariantAnnotation, GenomeNexusAPI } from 'cbioportal-frontend-commons';
 import { Mutation } from 'shared/api/generated/CBioPortalAPI';
 import LazyMobXCache, { CacheData } from 'shared/lib/LazyMobXCache';
 import AppConfig from 'appConfig';
 
 export type GenomeNexusCacheDataType = CacheData<VariantAnnotation>;
 
+// fetch function for GRCh37
```

Comment:
we should really rename this function.  fetch is too generic and should be reserved for the built in fetch function.